be/src/formats/csv/csv_reader.h

```diff
@@ -174,7 +174,7 @@ class CSVReader {
 
     virtual ~CSVReader() = default;
 
-    Status next_record(Record* record);
+    virtual Status next_record(Record* record);
```

Comment:
why change this, it will introduce virtual function call overhead.

Reply:
It's because we don't use parent class? So there will be no virtual func call overhead? 

If so, no need to make this function virtual.

```
        Status status = down_cast<HdfsScannerCSVReader*>(_reader.get())->next_record(&record);
```

Reply:
removed